lms/djangoapps/courseware/views/views.py

```diff
@@ -1078,7 +1078,7 @@ def dates(request, course_id):
 
     context = {
         'course': course,
-        'course_date_blocks': [block for block in course_date_blocks if block.title != 'current_datetime'],
+        'course_date_blocks': [block for block in course_date_blocks],
```

Comment:
Could this just become `'course_date_blocks': course_date_blocks` or if you really just want to convert it to a list, `'course_date_blocks': list(course_date_blocks)`?